### DISCUSSION TITLE
Metrics not visible for Micronaut application

### DISCUSSION THREAD

**Discussion by skundu1234** [apache/skywalking#9428] [General]
We have implemented Springboot and Micronaut applications with skywalking Java agent.Springboot we are getting endpoints and traces but dont see any value for micronaut application.Is there any upcoming plugin to support this micronaut application or it will be added in new Skywalking java agent.
Please let us know.

**Comment by wu-sheng** [comment]
AFAIK, the committer team has no one working on Micronaut relative ecosystem. 
The plugins are contributed by the community who are using various tech stacks. So, you may need to follow the plugin dev doc, and try to contribute plugin(s) to cover this case.

**Comment by pg-yang** [comment]
If nobody work on Micronaut ,Could I can try this?

**Reply by wu-sheng** [reply]
Yes, you could follow the dev doc, and contribute this to the community with plugin test scenario.

**Comment by skundu1234** [comment]
@pg-yang  It will be very helpful if you try and deploy.Thanks in advance

**Comment by pg-yang** [comment]
@skundu1234  I open a issue about this discussion. you can follow  at #9429

**Reply by skundu1234** [reply]
Thank you very much

**Comment by skundu1234** [comment]
Thank you very much @pg-yang  and @wu-sheng for your help to deploy micronaut plugin jar.May I know the probable date you  guys are planning to release java-8.12.0

**Reply by wu-sheng** [reply]
Usually, the Java agent gets released every 2-3 months. If this PR gets merged, this would be included in Sep.'s release.

**Reply by skundu1234** [reply]
ok Thanks @wu-sheng for that

**Comment by skundu1234** [comment]
@pg-yang  can you please confirm that profiling is working for your application
